pkg/flavor/kubernetes/flavor.go

```diff
@@ -787,6 +787,8 @@ func (flavor *Flavor) ListVolumeAttachments() (*storage_v1.VolumeAttachmentList,
 	if err != nil {
 		return nil, err
 	}
+	log.Trace("VOLUME ATTACHMENT LIST:", vaList)
+	log.Tracef("VOLUME ATTACHMENT LIST:%+v", vaList)
```

Comment:
What is the purpose of the first one?

Reply:
These lines were for debugging purposes. Will remove this.